services/movie_session.py

```diff
@@ -42,3 +42,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+def get_taken_seats(movie_session_id):
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+    taken_seats = []
+    for ticket in tickets:
+        taken_seats.append({'row': ticket.row, 'seat': ticket.seat})
+    return taken_seats
```

Comment:
The `get_taken_seats` function can be optimized using list comprehension. You can replace the loop with: `return [{'row': ticket.row, 'seat': ticket.seat} for ticket in tickets]`.